### DISCUSSION TITLE
Page Limit

### DISCUSSION THREAD

**Discussion by SimplitServices** [docusealco/docuseal#326] [Q&A] [answered]
Hello,

Is there a page limit on the number of pages you can upload in self-hosted docuseal ?

I'm unable to upload more than 16 pages. No errors encountered. Docuseal simply uploads the first 16 pages.
Tried looking up for an answer online to no avail

Thanks in advance

**Comment by omohokcoj** [comment]
@SimplitServices are you using the default docuseal/docuseal docker image? it should work ok there

**Reply by SimplitServices** [reply]
Yes It is the default docker image

**Reply by omohokcoj** [reply]
@SimplitServices i just tried it myself and it works - can you share the app logs?

**Reply by SimplitServices** [reply]
I've even tried uploading a blank freshly created pdf document with more than 16 pages, in case the issue was with my original pdf doc
Same result...

**Reply by SimplitServices** [reply]
> @SimplitServices i just tried it myself and it works - can you share the app logs?

I'm not very well versed. Can you guide please ?

**Reply by omohokcoj** [reply]
@SimplitServices can you please share the app logs - docker logs [app-name]

**Reply by SimplitServices** [reply]
I ran sudo docker logs docuseal
Not sure if this was what you asked me, but here is what I got..
[logs.txt](https://github.com/user-attachments/files/16414691/logs.txt)

**Reply by SimplitServices** [reply]
Ignore the previous reply
I think this is the correct logs
The issue was encountered on Friday 26/07. I can repeat the issue if required.
[logs.txt](https://github.com/user-attachments/files/16414738/logs.txt)

**Reply by SimplitServices** [reply]
And here specific logs with the issue reproduced just now :
Freshly created pdf of 21 pages
Only 16 pages uploaded..
[logs.txt](https://github.com/user-attachments/files/16415033/logs.txt)

**Reply by SimplitServices** [reply, marked as answer]
Hello @omohokcoj ,

I think I have the answer. I'm running the onpremise 1.5.2 release. (released 15 Apr 24)
Its a bug corrected on the 1.5.3 release (https://github.com/docusealco/docuseal/releases/tag/1.5.3)

Thanks anyway,
Regards,
Tasneem
